test/ir_Corona_test.cpp

```diff
@@ -1086,6 +1189,12 @@ TEST(TestDecodeCoronaAc, RealExample2) {
   ASSERT_EQ(decode_type_t::CORONA_AC, irsend.capture.decode_type);
   ASSERT_EQ(kCoronaAcBitsShort, irsend.capture.bits);
   EXPECT_STATE_EQ(expectedState_A1, irsend.capture.state, irsend.capture.bits);
+  // this is special, but showing what it might be
+  EXPECT_EQ(
+      "Power: On, Power Toggle: Off, Mode: 0 (Heat), Temp: 16C, "
+      "Fan: 2 (Medium), Swing(V) Toggle: Off, Econo: Off, "
+      "On Timer: Off, Off Timer: Off",
+      IRAcUtils::resultAcToString(&irsend.capture));
```

Comment:
```
ir_Corona_test.cpp:1197: Failure
Expected equality of these values:
  "Power: On, Power Toggle: Off, Mode: 0 (Heat), Temp: 16C, " "Fan: 2 (Medium), Swing(V) Toggle: Off, Econo: Off, " "On Timer: Off, Off Timer: Off"
    Which is: "Power: On, Power Toggle: Off, Mode: 0 (Heat), Temp: 16C, Fan: 2 (Medium), Swing(V) Toggle: Off, Econo: Off, On Timer: Off, Off Timer: Off"
  IRAcUtils::resultAcToString(&irsend.capture)
    Which is: "Power: On, Power Toggle: Off, Mode: 0 (Heat), Temp: 16C, Fan: 2 (Medium), Swing(V) Toggle: Off, Econo: Off, On Timer: Off, Off Timer: 18:59"
```

Reply:
Thanks for the ping,
explanation: `setRaw` is called with state, that only has 7 bytes populated, and then it defaults to trying to use 21 bytes.
Windows seems to have clean memory and this issue did not pop up there, reproduced with random values on the Linux box.

will add usage of nbits from result `ac.setRaw(decode->state, decode->bits / 8);` in `decodeToState`

Reply:
hmm something else is going wrong here, not sure what.

Reply:
You got half of the locations that call(ed) `Corona.setRaw()`, you also need to do it in `resultToString()` (or what ever it is called.)